go/cmd/dolt/commands/tblcmds/rm.go

```diff
@@ -84,23 +84,26 @@ func (cmd RmCmd) Exec(ctx context.Context, commandStr string, args []string, dEn
 	}
 
 	working, verr := commands.GetWorkingWithVErr(dEnv)
+	if verr != nil {
+		return exitWithVerr(verr)
+	}
 
-	if verr == nil {
-		verr := commands.ValidateTablesWithVErr(apr.Args(), working)
```

Comment:
This was shadowing the outer `verr` and causing the assignment on line 92 to be ineffectual. I could have fixed it by just changing `:=` to `=` but wanted to unindent the logic anyway.